Jakefile

```diff
@@ -31,6 +31,19 @@ task('minify', function () {
   console.log('Minification completed.');
 });
 
+task('doc', function () {
+  exec('./node_modules/.bin/rimraf out');
```

Comment:
Jake has a builtin for this: http://jakejs.com/docs#file-utils (`jake.rmRf`). Or now can use the `exec` defined in the Jakefile.